### DISCUSSION TITLE
Could not connect to postgres

### DISCUSSION THREAD

**Discussion by FyZzyss** [apache/airflow#16083] [Q&A] [answered]
I created a postgres connection and pass it in the code, but I get a connection error.
```
[2021-05-26 09:33:12,356] {taskinstance.py:1455} ERROR - could not connect to server: Connection refused
Is the server running on host "localhost" (127.0.0.1) and accepting
TCP/IP connections on port 5432?
could not connect to server: Cannot assign requested address
Is the server running on host "localhost" (::1) and accepting
TCP/IP connections on port 5432?
```

This database is 100% accessible, because I work with it through pycharm and asyncpg.

My connection in AirFlow:
![image](https://user-images.githubusercontent.com/38393787/119638668-8cc64600-be1f-11eb-84c5-94bf80444c6b.png)

Connection in code:
 ```
PostgresHook(postgres_conn_id="commerce_catalog_postgres").insert_rows(
                'public.arts',
                records
            )
```

What could be the problem?

**Comment by FyZzyss** [comment, marked as answer]
The problem is that air flow is running in docker and is looking at its localhost

**Reply by JanadaJayathilaka** [reply]
what is the solution for that
